openc3/lib/openc3/utilities/bucket_file_cache.rb

```diff
@@ -87,7 +87,7 @@ def retrieve(client = @bucket, uncompress = true)
           # Try to retrieve the file three times
           retry_count += 1
           raise err if retry_count >= 3
-          Logger.warn("Error retrieving log file from bucket - retry #{retry_count}: #{@bucket_path}\n#{err.formatted}")
+          OpenC3::Logger.warn("Error retrieving log file from bucket - retry #{retry_count}: #{@bucket_path}\n#{err.formatted}")
```

Comment:
Hmm, did this just crash the retry and thus no retries?

Reply:
Yep no retries.